src/clients/locations/FeatureServiceClient.js

```diff
@@ -6,20 +6,28 @@ const trackDependency = require('../appinsights/AppInsightsClient').trackDepende
 
 const apiUrlBase = process.env.FORTIS_FEATURE_SERVICE_HOST;
 
-function formatIdsUri(ids) {
-  return `${apiUrlBase}/features/id/${ids.map(encodeURIComponent).join(',')}?include=bbox`;
+function formatIdsUri(ids, include) {
+  let uri = `${apiUrlBase}/features/id/${ids.map(encodeURIComponent).join(',')}`;
```

Comment:
What about using `const` for `uri`? (there's a couple of other instances where there's `let uri`)